app/views/users/index.haml

```diff
@@ -26,7 +28,7 @@
       %th
       %th
   %tbody
-    - @users.each do |user|
+    - @active.each do |user|
```

Comment:
Not a terrible idea, but if it were me I'd just use `@users` in both views. They're both collections of users--information about what kinds of users they are can remain in the controller.

Reply:
For an alternative approach, I suggested not having two views.